ui/src/views/Settings.vue

```diff
@@ -19,26 +19,25 @@
       <div class="bx--col-lg-16">
         <cv-tile :light="true">
           <cv-form @submit.prevent="configureModule">
-            <cv-text-input
+            <NsTextInput
               :label="$t('settings.collabora_fqdn')"
-              placeholder="collabora.example.org"
-              v-model.trim="host"
-              class="mg-bottom"
+              :placeholder="$t('settings.placeholder_collabora_domain')"
+              v-model="host"
```

Comment:
Let's keep `.trim`, so whitespace is trimmed automatically from FQDN
```suggestion
              v-model.trim="host"
```